csharp/OnnxRuntime.CSharp.proj

```diff
@@ -16,6 +16,7 @@ CMake creates a target to this project
     <NugetExe Condition= " '$(NugetExe)' == '' ">nuget</NugetExe>
     <TargetArchitecture Condition=" '$(TargetArchitecture)' == '' ">x64</TargetArchitecture>
     <IsReleaseBuild Condition=" '$(IsReleaseBuild)' == '' ">false</IsReleaseBuild>
+	<IsLinuxBuild Condition=" '$(IsLinuxBuild)' == '' ">libonnxruntime.so</IsLinuxBuild>
```

Comment:
Should IsLinuxBuild just be a boolean ? Just like IsReleaseBuild ? That way it will be consistent and indicative of the variable name. (Personally, I don't think we need to make the shared library name configurable (i.e.) it is always going to be libonnxruntime.so I think)

Reply:
agreed that it should be boolean, rather than specifying name of the shared lib.